core/src/ibc/client_02/mod.rs

```diff
@@ -15,11 +15,14 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pub mod client;
+mod commitment_23;
 mod manager;
 pub mod types;
+
```

Comment:
Please remove the unrelated code change.
Please check your PR once before you request review.

Reply:
I'll fix the new line in line 25, but I think here is a valid change I think. Don't we separate `mod` and `use`? @majecty

Reply:
Please fix it in a new commit.

Reply:
You mean a commit that just adds a single new line?

Reply:
Yes. That is what I mean.

Reply:
@majecty fixed.

Reply:
Please use more descriptive commit message.
Like this: "Add a new line between mod statement and use statement.

Reply:
@majecty done!